device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnectionTest.java

```diff
@@ -308,6 +310,53 @@ public void openEstablishesConnectionUsingCorrectConfig() throws IOException, Tr
         };
     }
 
+    @Test
+    public void openEstablishesConnectionUsingModelIdSetsCorrectPreviewApiVersion() throws IOException, TransportException
```

Comment:
Add another test for scenario where you don't set the model Id

Reply:
If you look at openEstablishesConnectionUsingCorrectConfig - this does not set model id and makes sure API version is IOTHUB_API_VERSION so it is already covered

Reply:
cool